### PR TITLE
♻️ Permissions/Scope update for logs

### PR DESCRIPTION
**Pull request by astroash**
**Changes:**
- update api.json to new structure with new permissions
- move old api.json to archive folder in `./docs`
- update logs permissions in permissions.seed.json
- remove permissions.json as we don't need two versions of this file!
- remove all migrations
- add `NOT NULL` sector/region_id for cb table to db build (previously added in a migration)
- update scopes with new logs permissions
- update scopes in tests

### CODE REVIEW

File: database/seeds/permissions.seed.json [hunk 1]
```diff
@@ -27,18 +27,21 @@
...
+    "user_details-sibling:write",
     "visit_activities-child:read",
     "visit_activities-own:delete",
     "visit_activities-own:read",
     "visit_activities-own:write",
     "visit_logs-child:read",
-    "visit_logs-own:read",
-    "visit_logs-own:write",
+    "visit_logs-child:write",
     "volunteer_logs-child:read",
```
Comment (eliascodes) [inline database/seeds/permissions.seed.json]: Don't we also need `write` and `delete` versions of `volunteer_logs-child`?

File: database/seeds/permissions.seed.json [hunk 2]
```diff
@@ -101,27 +102,32 @@
       "visit_activities-own:delete",
       "visit_activities-own:read",
       "visit_activities-own:write",
-      "visit_logs-own:read",
-      "visit_logs-own:write",
-      "volunteer_logs-own:read"
+      "visit_logs-child:read",
+      "visit_logs-child:write",
+      "volunteer_logs-child:read"
```
Comment (eliascodes) [inline database/seeds/permissions.seed.json]: Write and delete logs?

**Comment by eliascodes** [comment]
I take it this hasn't been merged with my branch yet? I haven't looked too closely at the changes to the volunteer logs files, because presumably the merge will act as a cross check between mine and your changes.

File: database/seeds/permissions.seed.json (hunk 1 above)
Comment (astroash) [reply database/seeds/permissions.seed.json]: Yup, I forgot org_admin are allowed to login to the volunteer app.

File: src/api/v1/community_businesses/volunteer_logs.ts [hunk 3]
```diff
@@ -107,8 +272,7 @@ const routes: Hapi.ServerRoute[] = [
       auth: {
```
Comment (astroash) [inline src/api/v1/community_businesses/volunteer_logs.ts]: @eliascodes does this route need the `-sibling` scope? The payload doesn't include userId's so I imagine this route if just for adding your own logs.

**Comment by astroash** [comment]
@eliascodes this has now been merged with your branch. I just have one question re the `/sync` route.

File: src/api/v1/community_businesses/volunteer_logs.ts (hunk 3 above)
Comment (eliascodes) [reply src/api/v1/community_businesses/volunteer_logs.ts]: It's not something I've yet implemented, but I think it was a feature that was supported by the old API

Comment (astroash) [reply src/api/v1/community_businesses/volunteer_logs.ts]: cool, lets leave it for now then.

File: src/api/v1/api.json [hunk 4]
```diff
@@ -335,8 +334,7 @@
     "POST": {
       "isImplemented": true,
       "permissions": {
-        "VOLUNTEER": "volunteer_logs-own:write",
-        "VOLUNTEER_ADMIN": ["volunteer_logs-sibling:write", "volunteer_logs-own:write"],
+        "VOLUNTEER_ADMIN": ["volunteer_logs-sibling:write"],
```
Comment (eliascodes) [inline src/api/v1/api.json]: This needs to retain the `own` scopes for both `VOLUNTEER_ADMIN` and `VOLUNTEER` users.

File: src/api/v1/api.json [hunk 5]
```diff
@@ -325,8 +325,7 @@
...
```
Comment (eliascodes) [inline src/api/v1/api.json]: This needs to retain the `own` scopes for both `VOLUNTEER_ADMIN` and `VOLUNTEER` users.

**Comment by eliascodes** [comment]
🙏 Let's hope this doens't break anything

**Pull request merged**